docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+---
+title: "Path algorithms"
+linkTitle: "Path algorithms"
+description: "Learn how to use algo.SPpaths and algo.SSpaths to solve queries"
+weight: 7
+---
+
+RedisGraph 2.10 introduced two new path-finding algorithms, or more accurately, minimal-weight, optionally bounded-cost, and optionally bounded-length path-finding algorithms, `algo.SPpaths` and `algo.SSpaths`.
+
+`algo.SPpaths` and `algo.SSpaths` can solve a wide range of real-world problems, where minimal-weight paths need to be found. algo.SPpaths finds paths between a given pair of nodes, while algo.SSpaths finds paths from a given source node. Weight can represent time, distance, price, or any other measurement. A bound can be set on another property (e.g., finding a minimal-time bounded-price way to reach from point A to point B). Both algorithms are performant and have low memory requirements.
+
+For both algorithms, you can set:
+
+* A list of relationship types to traverse (`relTypes`).
+
+* The relationships' property whose sum you want to minimize (`weight`).
+
+* A optional relationships' property whose sum you want to bound (`cost`) and the optional bound (`maxCost`).
+
+* An optional bound on the path length - the number of relationships along the path (`maxLen`).
+
+* The number of paths you want to retrieve: either all minimal-weight paths (`pathCount` is 0), a single minimal-weight path (`pathCount` is 1), or _n_ minimal-weight paths with potentially different weights (`pathCount` is _n_).
+
+This topic explains which problems you can solve using these algorithms and demonstrates how to use them.
+
+Let's start with the following graph.
+
+![Road network](../images/road_network.png)
+
+This graph represents a road network with 7 cities (A, B, C, and so on) and 11 one-way roads. Each road has a distance (say, in kilometers) and trip time (say, in minutes).
```

Comment:
In case we're addressing US developers consider switching from KM to miles

Reply:
We're not addressing a specific geography.